### PR TITLE
Delete [SuppressMessage(...)] attributes for old/defunct/unused rules

### PR DESCRIPTION
**Pull request by stephentoub**

### CODE REVIEW

File: src/libraries/Common/src/System/Drawing/KnownColor.cs [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Licensed to the .NET Foundation under one or more agreements.
+﻿// Licensed to the .NET Foundation under one or more agreements.
```
Comment (jkotas) [inline src/libraries/Common/src/System/Drawing/KnownColor.cs]: Is this adding back BOMs?

Comment (stephentoub) [reply src/libraries/Common/src/System/Drawing/KnownColor.cs]: Probably. Will fix.

**Comment by stephentoub** [comment]
@jkotas, you ok with this?

**Review by jkotas — APPROVED** [review]

**Comment by stephentoub** [comment]
Thanks

**Pull request merged**